src/sse/sse.controller.ts

```diff
@@ -16,12 +18,12 @@ export class SseController {
         return this.sseService.observable;
     }
 
-    @Post('sse/publish')
-    @ApiOperation({
-        description: 'Publishes an event to the event stream',
-    })
-    @ApiOkResponse({ description: 'Publish an event' })
-    publish(@Body() params: MessageEvent): void {
-        this.sseService.emit(params);
-    }
+    // @Post('sse/publish')
```

Comment:
Publish event disabled?

Reply:
just disables the api endpoint to publish events. We don't use it for now. We might need it at some point to keep clients in sync so keeping it commented out for now.